### PR TITLE
Add the ability to specify an RPM url, per #1

### PR DESCRIPTION
**Pull request by ryansb**

### CODE REVIEW

**Comment by ryansb** [comment]
@chrisgilbert can you please review this change?

File: Vagrantfile [hunk 1]
```diff
@@ -17,9 +17,6 @@ Vagrant.configure('2') do |config|
 
   config.vm.provision :chef_solo do |chef|
     chef.json = {
-      :riemann => {
-        :version => '0.2.4'
```
Comment (chrisgilbert) [inline Vagrantfile]: Is this used anywhere to override the default?  It looks like you took it out of the chef recipe, so are still using the 0.2.10 package?

**Comment by chrisgilbert** [comment]
Hi Ryan, sorry for the delay, I have been off on honeymoon.  Just the one line comment, as I couldn't see you overriding the package anywhere with that version (since you removed the package version string concat, it doesn't put the version in anywhere I can see now).

File: Vagrantfile (hunk 1 above)
Comment (ryansb) [inline Vagrantfile]: No, it's not used anywhere I'm aware of. I switched to using the package URL, since that's the only place the version number is used.

**Pull request merged**